index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Richard Mbuti</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style></style>
+  </head>
+  <body>
+    <header class="topHeader">
+      <a href="#" class="logo">RK</a>
+      <nav class="menu">
+        <a href="#" class="toggleNav">
+          <img src="img/Union.png" alt="menu_button" />
+        </a>
+      </nav>
+    </header>
+    <br />
+    <br />
+    <section class="backgroundImg">
+      <div class="Personal_info">
+        <h1 class="introName">I'm Richard Mbuti Glad to see you!</h1>
+        <p class="intro">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <h6 class="connect">LET'S CONNECT</h6>
+      </div>
+      <div class="my_links">
```

Comment:
:red_circle: Kindly make sure the space between these elements match the Figma Design

Your Design
![final](https://user-images.githubusercontent.com/26736582/205443133-863cdfb2-874c-4ae6-8891-e6353d57c062.png)

Figma Design
![figma](https://user-images.githubusercontent.com/26736582/205443074-6f1a55da-4c90-41a1-a026-e280130fb0b2.png)